tensorflow/buildspec-tf1.yml

```diff
@@ -95,7 +95,7 @@
       docker_file: !join [ docker/, *VERSION, /, *DOCKER_PYTHON_VERSION, /Dockerfile., *DEVICE_TYPE ]
       context:
         <<: *TRAINING_CONTEXT
-      BuildTensorflowGPUTrainPy2DockerImage:
+    BuildTensorflowGPUTrainPy2DockerImage:
         <<: *TRAINING_REPOSITORY
```

Comment:
Should these lines also be adjusted for indentation? (To move ahead of 2 spaces)